metabase/metabase
# add percentages to sankey tooltip

[Slack convo](https://metaboat.slack.com/archives/C064QMXEV9N/p1734361812223819?thread_ts=1734023496.547399&cid=C064QMXEV9N)

### Description

Extends sankey tooltips with more data. Node tooltips show all output values with percentages. Edge tooltip show the the same information but highlight the edge value.

### How to verify

- Create a Sankey chart
- Hover nodes and edges
- Ensure the tooltip show correct values

### Demo

<img width="778" alt="Screenshot 2024-12-13 at 10 53 02 PM" src="https://github.com/user-attachments/assets/7651cacb-bea7-4090-8ca6-25165a5774da" />

### Checklist
